docs/develop/create-addin-commands.md

```diff
@@ -146,55 +96,50 @@ The following is an example of the **\<FunctionFile\>** element.
+
+```js
+// Initialize the Office Add-in.
+Office.onReady(() => {
+  // If needed, Office.js is ready to be called
+});
+
+// The command function.
+async function highlightSelection(event) {
+
+    // Implement your custom code here. The following code is a simple Excel example.  
+    try {
+          await Excel.run(async (context) => {
+              const range = context.workbook.getSelectedRange();
+              range.format.fill.color = "yellow";
+              await context.sync();
+          });
+      } catch (error) {
+          // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+          console.error(error);
+      }
+
+    // Calling event.completed is required. event.completed lets the platform know that processing has completed.
+    event.completed();
+}
+
+// You must register the function with the following line.
+Office.actions.associate("highlightSelection", highlightSelection);
+
 ```
 
-> [!IMPORTANT]
-> The call to **event.completed** signals that you have successfully handled the event. When a function is called multiple times, such as multiple clicks on the same add-in command, all events are automatically queued. The first event runs automatically, while the other events remain on the queue. When your function calls **event.completed**, the next queued call to that function runs. You must implement **event.completed**, otherwise your function will not run.
+#### Outlook notifications
 
-## Step 6: Add ExtensionPoint elements
+When an add-in needs to provide status updates, such as progress indicators or error messages, it must do so through the [notification APIs](/javascript/api/outlook/office.notificationmessages). The processing for the notifications must also be defined in a separate HTML file that is specified in the `FunctionFile` node of the manifest.
 
-The **\<ExtensionPoint\>** element defines where add-in commands should appear in the Office UI. You can define **\<ExtensionPoint\>** elements with these **xsi:type** values.
+### Step 4: Add ExtensionPoint elements
 
-- **PrimaryCommandSurface**, which refers to the ribbon in Office.
+The [**\<ExtensionPoint\>** element](/javascript/api/manifest/extensionpoint) defines where add-in commands should appear in the Office UI. You can define **\<ExtensionPoint\>** elements with these **xsi:type** values.
 
-- **ContextMenu**, which is the shortcut menu that appears when you right-click in the Office UI.
+- **PrimaryCommandSurface**: The ribbon in Office.
+- **ContextMenu**: The shortcut menu that appears when you right-click in the Office UI.
```

Comment:
These values only apply to Excel, OneNote, PowerPoint, and Word. Outlook extension points are listed in [Extension points for Outlook](https://learn.microsoft.com/javascript/api/manifest/extensionpoint#extension-points-for-outlook). Consider linking to the article for product-specific values.

Reply:
Good catch. I'll remove the text and let that ExtensionPoint link do the heavy lifting.